app/src/main/java/org/oppia/app/help/HelpActivity.kt

```diff
@@ -16,4 +19,16 @@ class HelpActivity : InjectableAppCompatActivity() {
     helpActivityPresenter.handleOnCreate()
     title = getString(R.string.menu_help)
   }
+
+  companion object {
+    fun createHelpActivityIntent(context: Context, profileId: Int?): Intent {
+      val intent = Intent(context, HelpActivity::class.java)
+      intent.putExtra(KEY_HOME_PROFILE_ID, profileId)
```

Comment:
This is not required in HELP

Reply:
@rt4914 I have done this implementation and also changed both the intent calls in NavigationDrawerFragmentPresenter because otherwise Administrator Controls option won't be visible when we move to Help Activity and then open nav. drawer.

Reply:
Replied above